conf/crick.config

```diff
@@ -1,23 +1,32 @@
-//Profile config names for nf-core/configs
+// Profile config names for nf-core/configs
 params {
   config_profile_description = 'The Francis Crick Institute CAMP HPC cluster profile provided by nf-core/configs.'
-  config_profile_contact = 'Harshil Patel (@drpatelh)'
-  config_profile_url = 'https://www.crick.ac.uk/research/platforms-and-facilities/scientific-computing/technologies'
+  config_profile_contact     = 'Harshil Patel (@drpatelh)'
+  config_profile_url         = 'https://www.crick.ac.uk/research/platforms-and-facilities/scientific-computing/technologies'
 }
 
+// Load Ensembl genomes available on CAMP
+includeConfig "${params.custom_config_base}/conf/genomes/crick/genomes.config"
+
 singularity {
-  enabled = true
+  enabled    = true
   autoMounts = true
 }
 
 process {
   executor = 'slurm'
+
+  withName:RSEM_CALCULATEEXPRESSION  {
+    cpus   = 12
+    memory = 72.GB
+    time   = 16.h
+  }

```

Comment:
This part seems to be pipeline specific.
Should be in a special `rnaseq/crick.config` file, no?

Reply:
Genius @MaxUlysse !

Reply:
Done in https://github.com/nf-core/configs/pull/198/commits/49d18f4fcccd50b2d27e38830afafd41834ac021 and  https://github.com/nf-core/configs/pull/198/commits/1ae378b65cdb787877d6351906b78ac0b6c5a429